src/groups/heisenberg.jl

```diff
@@ -26,10 +26,11 @@ function HeisenbergGroup(n::Int; parameter::Symbol=:type)
     return HeisenbergGroup{typeof(size)}(size)
 end
 
+is_embedded_manifold(::HeisenbergGroup) = true
```

Comment:
Just as a general remark, this function might need a case where it depends on the point type? Some representations might be embedded, others not. 

Though that might not affect this specific case, we at least should have that in mind and handle it similar to other cases like rand.

Reply:
That's a tricky point. It might indeed have to depend on the point type. I will think about it.